### DISCUSSION TITLE
Portfolio/Token Timeline up to an year not only month

### DISCUSSION THREAD

**Discussion by dlt-green** [iotaledger/firefly#1529] [Features / Enhancements]
## Description

It would be nice to get a longer timeline for the portfolio/token amount up to an year in firefly.

## Motivation

would be nice for hodler :P, not every person is a trader :-)

![Unbenannt](https://user-images.githubusercontent.com/89119285/129836427-e5118253-b1e0-44cd-bdb4-9a1afa2164f6.png)

**Comment by nicole-obrien** [comment]
Some thoughts if we pick this up: 
- It would be nice to have this dynamic based on the data available. i.e. if we have less that 1 month data don't show the 1 year in drop down etc.
- Add additional options 3 months, and all time.
